classes/opponentSpawner.lua

```diff
@@ -69,9 +91,13 @@ function opponentSpawner:draw()
 			local originX = next.SPAWN_X or (ARENA_WIDTH / 2)
 			local originY = next.SPAWN_Y or (ARENA_HEIGHT / 2)
 
-			love.graphics.circle("line", originX, originY, self.spawningTimeRemaining * 1200)
+			love.graphics.circle("line", originX, originY, (self.spawningTimeRemaining * 30) ^ 2)
 
 			love.graphics.setLineWidth(prevWidth)
+
+			love.graphics.setFont(fonts.large)
+			love.graphics.printf(self.introText[self.index], 200, ARENA_HEIGHT / 4, ARENA_WIDTH - 400, "center")
+			love.graphics.setFont(fonts.small)
```

Comment:
Also, don't worry about doing this. Rather, just set the font size each time you want to render text, don't assume the previous size